kale/utils/distance.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
```

Comment:
Doc at the top.

Reply:
I have added a docstring at the top of the file.